2020/04/20 20:12:19 [error] 24483#24483: *1 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET / HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/", host: "polls.cinemanager.co.nz"
2020/04/20 20:12:21 [error] 24483#24483: *1 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
2020/04/20 20:17:44 [error] 24483#24483: *7 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET / HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/", host: "polls.cinemanager.co.nz"
2020/04/20 20:17:44 [error] 24483#24483: *7 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
2020/04/20 20:25:07 [error] 24483#24483: *11 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET / HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/", host: "polls.cinemanager.co.nz"
2020/04/20 20:25:08 [error] 24483#24483: *11 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
2020/04/20 20:26:38 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET / HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/", host: "polls.cinemanager.co.nz"
2020/04/20 20:26:39 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
2020/04/20 20:27:25 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET / HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/", host: "polls.cinemanager.co.nz"
2020/04/20 20:27:26 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
2020/04/20 20:27:43 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET / HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/", host: "polls.cinemanager.co.nz"
2020/04/20 20:27:44 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
2020/04/20 20:27:48 [error] 24482#24482: *17 connect() to unix:/run/polls_django.sock failed (111: Connection refused) while connecting to upstream, client: 125.239.175.5, server: polls.cinemanager.co.nz, request: "GET /favicon.ico HTTP/1.1", upstream: "http://unix:/run/polls_django.sock:/favicon.ico", host: "polls.cinemanager.co.nz", referrer: "http://polls.cinemanager.co.nz/"
